### DISCUSSION TITLE
Closure Visualizer

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/runtime#50486] [Ideas]
Sometimes closures may capture more variables than intended.

It would be really nice if there was a "Closure Visualizer" in Visual Studio so that I could move my mouse over a lambda and see the variables that are captured in the closure.

**Comment by stephentoub** [comment]
That already exists:
![image](https://user-images.githubusercontent.com/2642209/113179335-7e8feb00-921d-11eb-8f04-63c623cd43e9.png)
